app/models/account_authorization_config/linked_in.rb

```diff
@@ -67,9 +67,18 @@ def authorize_options
 
   def scope
     if login_attribute == 'emailAddress'.freeze
-      'r_basicprofile r_emailaddress'.freeze
+      'r_fullprofile r_emailaddress w_share'.freeze
```

Comment:
This is my bad, but we don't want to request w_share privileges.  Can you remove that from here and below?